packages/test-utils/src/testUtils/appWrappers.tsx

```diff
@@ -226,7 +231,7 @@ export function wrapInTestApp(
  * @public
  */
 export async function renderInTestApp(
-  Component: ComponentType | ReactNode,
+  Component: ComponentType<PropsWithChildren<{}>> | ReactNode,
```

Comment:
This is actually not technically necessary, albeit not exactly harmful either. The test renderer just wants to render any component, it doesn't care about adding children to it.